src/lib/core/index.js

```diff
@@ -1,12 +1,55 @@
 import React from "react";
 import { withMutations } from "../mutate";
 import Frame from "../frame";
+import { connect } from "react-redux";
 import { Themed } from "../theme";
+import { setToast } from "../../redux/actions";
+import { ToastContainer, toast } from "react-toastify";
+import "react-toastify/dist/ReactToastify.min.css";
 
-const Core = () => (
-  <Themed>
-    <Frame />
-  </Themed>
-);
+class Core extends React.Component {
+  constructor(props) {
+    super(props);
+    this.props.dispatch(setToast("do you like cats?", "info"));
+  }
 
-export default withMutations(Core);
+  componentWillReceiveProps(nextProps) {
+    switch (nextProps.toast.type) {
+      case "error":
```

Comment:
Not really a big deal, but instead of a switch, what if you did:

``` js 
if (nextProps.toast.type) {
 toast[nextProps.toast.type](nextProps.toast.message);
 return;
} 

toast(nextProps.toast.message)

```

Reply:
Benefits: 
1. Less code
2. Incorrect inputs will error instead of being hidden (which makes bugs more visible) and therefore easily found.